content/docs/command-reference/exp/gc.md

```diff
@@ -5,8 +5,8 @@ Remove unnecessary experiments from the <abbr>project</abbr>.
 ## Synopsis
 
 ```usage
-usage: dvc exp gc [-h] [-q | -v] [-w]
-                  [-a] [-T] [-A] [--queued] [-f]
+usage: dvc exp gc [-h] [-q | -v] [-w] [-a] [-T] [-A]
+                  [--date <commit_date>] [--queued] [-f]
```

Comment:
Actually we had purposefully separated the lines, it's not a typo.
```suggestion
usage: dvc exp gc [-h] [-q | -v] [-w]
                  [-a] [-T] [-A] [--date <commit_date>]
                  [--queued] [-f]
```
Committing